browser/brave_stats/brave_stats_updater.cc

```diff
@@ -43,6 +43,10 @@
 namespace brave_stats {
 
 namespace {
+
+BraveStatsUpdater::StatsUpdatedCallback g_testing_stats_updated_callback;
+BraveStatsUpdater::StatsUpdatedCallback g_testing_stats_threshold_callback;
```

Comment:
Any reason not to declare this global variables `static`?

Reply:
Ah, ok... they're in an anonymous namespace. Forget my suggestion and sorry for the noise.